packages/plugin-flex/src/__tests__/commands/flex/plugins/deploy.test.ts

```diff
@@ -202,7 +208,7 @@ describe('Commands/FlexPluginsDeploy', () => {
       Version: deployResult.nextVersion,
       PluginUrl: deployResult.pluginUrl,
       Private: !deployResult.isPublic,
-      Changelog: defaultChangelog,
+      Changelog: 'sample%20changlog',
```

Comment:
This is wrong - why is space being encoded? Unless you change the API to decode this, you will be storing this in the DB which means customers are actually going it this way.,

Reply:
This change is regarding the issue FLEX-530. It's a security issue that was raised which said plugins CLI needs to ensure that there is no way to inject url params when passing values to the command flags/args

Reply:
Have you confirmed plugins-api will properly decode this? How is this going to be saved and how is the user then going to see this? When they query the API, are they gonna see `sample changelog` OR are they gonna see `sample%20changelog`?

Reply:
@pragya-arora Please confirm this behaviour

Reply:
I have checked that plugins-api decodes this properly. The user is going to see `sample changelog` (NOT `sample%20changelog`). 
For example, for the following command:
`twilio flex:plugins:deploy --major --changelog "Adding plugin 14 for testing" --description "Test plugin 14" -l debug`
We can see in the following debug logs screenshot that the changelog in request data has been encoded and response data contains decoded changelog:
![Screenshot 2022-09-26 at 1 43 18 PM](https://user-images.githubusercontent.com/103407403/192227845-a44bef5c-6fd8-41f9-873c-08a46b6f8413.png)
In following screenshot of Plugin dashboard from Admin Plugin, we can see that the changelog is decoded as well: 
![Screenshot 2022-09-26 at 1 44 49 PM](https://user-images.githubusercontent.com/103407403/192228097-07fdc19b-ff55-4477-8802-47ac069eed8f.png)